source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/EverythingBagel.json

```diff
@@ -118,12 +119,14 @@
 					"id": "ColumnSet_id",
 					"spacing": "large",
 					"separator": true,
+					"rtl":true,
```

Comment:
`rtl` is not supported on ColumnSet in JS (it is on Column). We can do either of two things:
- Change JS to match this
- Change this to match JS

I clearly prefer the latter.

Reply:
Done